tests/ignite/contrib/handlers/test_base_logger.py

```diff
@@ -103,7 +103,7 @@ def test_attach():
     n_epochs = 5
     data = list(range(50))
 
-    def _test(event, n_calls):
+    def _test(event, n_calls, kwargs):
```

Comment:
That's wrong, check here: https://github.com/pytorch/ignite/pull/2034/checks?check_run_id=2763600607#step:12:1608
Please before pushing the commits try to run the tests locally, that will save you a lot of time, to run the tests, check here: https://github.com/pytorch/ignite/blob/master/CONTRIBUTING.md#run-tests

Reply:
Thanks for this catch!

Reply:
Make sure please to fix the errors and run the tests locally